acceptance/tests/pxp-module-puppet/negative/attempt_puppet_flag_whitelist_bypass.rb

```diff
@@ -0,0 +1,58 @@
+require 'pxp-agent/test_helper.rb'
+
+test_name 'C100245 - attempt bypass of flag whitelist with tab character' do
+
+  confine :except, :platform => ['windows']
```

Comment:
Why not Windows? Would be useful to add a comment explaining the exclusion.

It could be valuable to see this test pass on all platforms - to ensure that the latest pxp-module-puppet script has been packaged